docs/_plugins/rhds.cjs

```diff
@@ -242,6 +291,11 @@ module.exports = function(eleventyConfig, { tagsToAlphabetize }) {
     }
   });
 
+  // custom-elements.json
+  eleventyConfig.on('eleventy.before', async function() {
+    allRelationships = await getAllRelationships();
+  });
+
```

Comment:
```suggestion
```
make put this in the 11ty data cascade instead, e.g. with [`addGlobalData`](https://www.11ty.dev/docs/data-global-custom/)